website/docs/tutorials/installation/00_overview.md

```diff
@@ -0,0 +1,30 @@
+---
+id: overview
+title: 00 - Installing Custom to Brain
+---
```

Comment:
This should be part of the introduction tutorial. The idea of the introduction tutorial is to be able to create, configure, deploy, and run a custom app that does little to nothing

Reply:
Sounds good.